providers/teams/pages/add.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { ChangeEvent, useEffect } from 'react'
+import dynamic from 'next/dynamic'
+import { gql } from 'apollo-boost'
+import { useRouter } from 'next/router'
+import { apolloClient } from '../lib/apollo'
+import cookie from 'js-cookie'
+import { decode } from 'jsonwebtoken'
+import { Team } from '../lib/common'
+
+/**
+ * Note: we're sending the variables to grahpql by the input's name.
+ */
+export const AddTeamPage = props => {
+    const router = useRouter()
+    const userId = getUserId()
+
+    useEffect(() => {
+        async function redirectIfHasTeam() {
+            const teamId = await hasTeam(userId)
+            if (teamId) {
+                router.replace(`/team/${teamId}`)
+            }
+        }
+        redirectIfHasTeam()
+    }, [])
+
+    async function onSubmit(event: ChangeEvent<HTMLFormElement>) {
+        event.preventDefault()
+        const formElement = document.forms[0]
+        const formData = new FormData(formElement)
+        const team: Team = {
+            name: formData.get('name').toString(),
+            description: formData.get('description').toString(),
+        }
+
+        const teamId = await insertTeams(team, userId)
+
+        router.replace(`/team/${teamId}`)
+    }
+
+    return (
+        <div>
+            <h1>Add Team Screen</h1>
+            <form onSubmit={onSubmit}>
+                <div>
+                    <label htmlFor="name">Team Name</label>
+                    <input type="text" name="name" />
+                </div>
+                <div>
+                    <label htmlFor="name">Description</label>
+                    <input type="text" name="description" />
+                </div>
+
+                <input type="submit" />
+            </form>
+        </div>
+    )
+}
+
+async function hasTeam(userId: string) {
+    // This both creates the team and puts the user into the team_members call.
```

Comment:
I guess this is an accident from copy-paste?

Reply:
yes, thanks for the heads up